core/whistle-1.0.0/src/wh_cache.erl

```diff
@@ -602,7 +602,11 @@ maybe_erase_changed(Db, 'database', _Id, Tab) ->
     erase_changed(Objects, [], Tab);
 
 maybe_erase_changed(Db, Type, Id, Tab) ->
-    MatchSpec = [{#cache_obj{origin = {'db', Db}, _ = '_'}
+    MatchSpec = [{#cache_obj{origin = {'database', Db}, _ = '_'}
```

Comment:
@macpie well, if you clear all the cache, you'll never have any cache problems, right ? check the origin bindings that were created. this changed will invalidate every cache entry for the same database.

for instance, 2 users with simple binding
{mydb, user, 1}
{mydb, user, 2}
would create 4 entries in cache 
{mydb, user, 1} + pointer{database, mydb}
{mydb, user, 2} + pointer{database, mydb}

your change will invalidate {mydb, user, 2} if {mydb, user, 1} is deleted.